practices/practice13/tasks.cpp

```diff
@@ -0,0 +1,122 @@
+#include <iostream>
+#include <string>
+#include <cctype>
+
+#define npos 4294967295
```

Comment:
? why we need it?  it is string::npos

Reply:
Because when the substring is not found, it returns the value of npos, but not of std::string::npos